MantidQt/CustomInterfaces/test/MuonAnalysisFitDataPresenterTest.h

```diff
@@ -148,6 +148,7 @@ class MuonAnalysisFitDataPresenterTest : public CxxTest::TestSuite {
   }
 
   void test_setAssignedFirstRun_singleWorkspace() {
+    setupGroupPeriodSelections();
```

Comment:
This just sets up what some methods will return when called, it doesn't test anything itself